### PR TITLE
Add a profile view/update route

### PR DESCRIPTION
**Pull request by Pomax**
ideally we have an `/api/pulse/myprofile` route that pulls up the profile of an authenticated user. However, I'm having a surprisingly hard time making that happen right now. 

@gideonthomas want to help with this?

closes https://github.com/mozilla/network-pulse-api/issues/170

### CODE REVIEW

File: pulseapi/urls.py [hunk 1]
```diff
@@ -25,18 +25,11 @@
     # new patterns
     url(r'^api/pulse/', include('pulseapi.users.urls')),
     url(r'^api/pulse/entries/', include('pulseapi.entries.urls')),
+    url(r'^api/pulse/profiles/', include('pulseapi.profiles.urls')),
     url(r'^api/pulse/tags/', include('pulseapi.tags.urls')),
     url(r'^api/pulse/issues/', include('pulseapi.issues.urls')),
     url(r'^api/pulse/helptypes/', include('pulseapi.helptypes.urls')),
     url(r'^api/pulse/creators/', include('pulseapi.creators.urls')),
-
```
Comment (Pomax) [inline pulseapi/urls.py]: this change will happen as separate PR once this PR's fixed up.

File: pulseapi/profiles/serializers.py [hunk 2]
```diff
@@ -12,3 +14,46 @@ class Meta:
...
+        required=False
+    )
+    issues = serializers.SlugRelatedField(
+        many=True,
+        slug_field='name',
+        queryset=Issue.objects,
+        required=True
+    )
+
+    twitter = serializers.models.URLField(max_length=2048)
```
Comment (gideonthomas) [inline pulseapi/profiles/serializers.py]: as opposed to `serializers.URLField`?

Comment (Pomax) [inline pulseapi/profiles/serializers.py]: man I don't even know - this was copy-pasta and clearly django doesn't care to check whether this even works in the slightest.

**Comment by gideonthomas** [comment]
looks good to me, just the one question above

**Pull request merged**